website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+
+But I will argue that the biggest shift initiated during 2018 is not code related;  this title should go to the empowerment that the React Native Community gained.
+
+Since the very beginning, Facebook allowed developers from all around the world to participate in this Open Source project; investing a few super active members of the community of responsibilities like owning the release cycle.
+
+And, this year, the members of the Core Contributors group took a few substantial steps towards making the whole community more able to shape the future of this awesome project: two repositories have been created throughout the year precisely with this goal in mind.
+
+## [`react-native-releases`](https://github.com/react-native-community/react-native-releases) 📬
+
+This repository, started around Feb, served the dual purpose of allowing everyone to keep up the new releases in a more collaborative manner and opened the conversation of what would be part of a certain release to whoever wanted to suggest a cherry-pick (like for [0.57.8](https://github.com/react-native-community/react-native-releases/issues/71) and all its previous versions).
+
+This has been the driving force behind moving away from a monthly release cycle, and the "long term support" approach currently used for version 0.57.x.
+
+Half of the merit in reaching these decisions goes to the other repository created this year:
+
+## [`discussions-and-proposals`](https://github.com/react-native-community/discussions-and-proposals) 🗣
+
+This other ‘meta’ repository, created a few months after its sister (in July), expanded on the idea of a more open environment for conversations on React Native; while, partially, this need was covered by a few issues labelled [`For Discussion`](https://github.com/facebook/react-native/labels/For%20Discussion) in the main repository, we wanted to give our own spin to the RFC approach that other libraries (like React) have.
+
+This experiment found immediately its role in the React Native lifecycle: even the FB team is now relying heavily on it to discuss what could [be improved in React Native](https://github.com/react-native-community/discussions-and-proposals/issues/64), and coordinate the efforts around the [Lean Core project](https://github.com/react-native-community/discussions-and-proposals/issues/6) - among other interesting conversations.
+
+But we are far from done.
+
+### What awaits ahead 🎓
+
+As explained above, over the past 6-8 months, the [React Native Community organisation](https://github.com/react-native-community) has become de-facto "officially endorsed by FB" - but it always lacked the formal structure that similar projects (like Flutter or Ionic) have for their plugin libraries. By enforcing a set of standards for all the packages/repos hosted in it, providing a common place for all the maintainers to help each other and providing quality code, this organisation can set the example for everyone in the larger developer community.
```

Comment:
I'm not really sure it makes sense here to specifically call out alternative frameworks here? Can we drop them?

I'm also not sure about "official endorsed by FB" in quotes. It sounds a bit odd. Could we just say "The React Native core contributors are taking more ownership over the development of React Native, and are collaborating with Facebook more than ever before"?

Reply:
I’ve changed it to be closer to your version, but I still think that it’s important to communicate that we are not ‘blind’ (as community around RN) to the other projects and that we have the sensibility to tell how and what is good, and use that to inspire positive change in our ‘world’.

Reply:
I understand what you are saying. I’m just pointing out that those projects aren’t the motivation for us to improve React Native, it’s purely coincidental that they have been doing similar things.

Reply:
I don't think this sentence really adds anything, what's the messaging you're after?